### PR TITLE
fix cm name + talk about using different channel

### PR DESCRIPTION
**Pull request by vaikas**
Clarify that if you use out of the box config, you get IMC channels, and clarify that if you want to use some other channel, you have to modify the configmap.

## Proposed Changes 

- Clarify that if you use out of the box config, you get IMC channels, and clarify that if you want to use some other channel, you have to modify the configmap.
-
-

### CODE REVIEW

File: docs/eventing/broker/README.md [hunk 1]
```diff
@@ -59,12 +59,13 @@ spec:
...
+`knative-eventing` namespace and is called `config-br-defaults`. Out of the box
 it comes configured to create
-[MT Channel Based Brokers](./mt-channel-based-broker.md). If you are using a
-different Broker implementation, you should modify the `ConfigMap`
+[MT Channel Based Brokers](./mt-channel-based-broker.md) as well In Memory
+Channels. If you are using a different Broker implementation, or want to use
+different channel implementation, you should modify the `ConfigMap`
 accordingly. You can read more details on how to use
-[default-br-config config map ](./config-br-defaults.md)
+[default-br-config configmap ](./config-br-defaults.md)
```
Comment (pierDipi) [inline docs/eventing/broker/README.md:68]:
```suggestion
[config-br-defaults `ConfigMap`](./config-br-defaults.md)
```

File: docs/eventing/broker/README.md [hunk 2]
```diff
@@ -59,12 +59,13 @@ spec:
 ### Creating a broker using defaults
 
 Knative Eventing provides a `ConfigMap` which by default lives in
-`knative-eventing` namespace and is called `default-br-config`. Out of the box
+`knative-eventing` namespace and is called `config-br-defaults`. Out of the box
 it comes configured to create
-[MT Channel Based Brokers](./mt-channel-based-broker.md). If you are using a
-different Broker implementation, you should modify the `ConfigMap`
+[MT Channel Based Brokers](./mt-channel-based-broker.md) as well In Memory
```
Comment (devguyio) [inline docs/eventing/broker/README.md:64]:
```suggestion
[MT Channel Based Brokers](./mt-channel-based-broker.md) as well as In Memory
```
nit

File: docs/eventing/broker/README.md [hunk 3]
```diff
@@ -59,12 +59,13 @@ spec:
...
+Channels. If you are using a different Broker implementation, or want to use
+different channel implementation, you should modify the `ConfigMap`
```
Comment (devguyio) [inline docs/eventing/broker/README.md:66]:
```suggestion
a different channel implementation, you should modify the `ConfigMap`
```
nit

**Comment by vaikas** [comment]
Thanks!! Updated as per suggestions 👍

**Pull request merged**